### ISSUE TITLE
Problem in a Tuning Setting  (SO DLC)

### ISSUE THREAD

**Issue by Albert0501** [Truinto/ONI-Modloader-SimpleMods#20]
There are tuning settings for rocket engine power, however the hydrogen engine power was actually affecting the radbolt engine, and there is no way to change hydrogen engine power

**Comment by Truinto** [comment]
I made the tuning settings on request. I checked almost non of them. I just exposed all of the tuning options that I found. Pretty sure there are even more. So if a setting doesn't work or is swapped that's either because I mixed them up or they are badly named.

I am willing to correct any mistake. Just tell me what exactly is wrong. In this case, what's the setting's exact name and on which building does it affect which parameter?

Note that I haven't played the DLC at all. I have no experience with the space stuff.

**Comment by Albert0501** [comment]
the ingame name: mod->config->tunning->Engine Power: Hydrogen Engine
name in Documents\Klei\OxygenNotIncluded\modsCustomizeBuildings.json: TuningEnginePowerLateStrong
I set the value 3 times as testing (125, 250, and 500)
However, in the game play, I click the "hydrogen engine" rocket module, under the "status" page, "rocketry" section, under the description of "rocket speed" it said "rocket engine power: 48" and not changing at all (by the way, I'm not setting any value to be 48, so this value must be unadjusted) 
Also I checked the "radbolt engine" rocket module (DLC new rocket engine), under the "status" page, "rocketry" section, under the description of "rocket speed" it said "rocket engine power: 500" (first 125, then 250), so this was affected by the setting.
And I didn't check other rocket engine module since they are not useful and I'm not building them.
I hope this info will help u, have a nice day.
